infrastructure.go

```diff
@@ -555,7 +556,8 @@ func (c *Client) reregisterNtfns() error {
 // ignoreResends is a set of all methods for requests that are "long running"
 // are not be reissued by the client on reconnect.
 var ignoreResends = map[string]struct{}{
-	"rescan": {},
+	"rescan":       {},
+	"rescanblocks": {},
```

Comment:
ACK. Makes sense. I wasn't 100% sure about this and thought it might be better to manually reissue the command since it can be quite a long search.